install.py

```diff
@@ -45,28 +45,35 @@
 
 print('%s[*]%s Probing the Linux subsystem...' % (Fore.GREEN, Fore.RESET))
 
-basedir, lxpath = probe_wsl()
+basedir, lxpath, bashpath = probe_wsl()
+#fix basedir to add LocalState\rootfs
```

Comment:
Missed space after # here and below. All python linters will mark this as an error.